README.md

```diff
@@ -42,6 +42,11 @@ ize tunnel up
 ize tunnel down
 ```
 
+### 6. Run application inside ECS container
+_If you need to manually run application in a ECS hosted docker container you can use the following command:
+```shell
+ize exec goblin -- bash --ecs-cluster testnut
```

Comment:
I'm not sure this command is valid. `--ecs-cluster` can't come after the command.
Also, `--ecs-cluster` is not a mandatory parameter (it gets derived automatically from `namespace` and `env`), so maybe omit it completely?